.evergreen/config.yml

```diff
@@ -200,6 +200,9 @@ functions:
             mv $i.new $i
           done
 
+          # Copy client certificate because symlinks do not work on Windows.
+          cp ${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem ${MONGO_ORCHESTRATION_HOME}/lib/client.pem
```

Comment:
do other drivers not need to do this? why is this specific to us?

Reply:
Drivers that test on windows do; I coped from Python [here](https://github.com/mongodb/mongo-python-driver/blob/master/.evergreen/config.yml#L642-L643).

One thing I found out when looking for this after hitting an error is that surprisingly few drivers actually test on Windows.